Automattic/wp-calypso
# VideoPress User Intent Onboarding: Add Sensei/video course card

Fix Automattic/greenhouse#1863

## Proposed Changes

This PR replaces the "single video upload" card with the sensei card in the VideoPress user intent onboarding.

## Testing Instructions

* Apply PR
* `yarn start`
* Go to `calypso.localhost:3000/setup/videopress/`
* ✅ You should see the new "video course" card replacing the "singe video upload" card
* Click on it
* ✅ It should display Sensei features
* Click on the "Discover sensei" button
* ✅ You should be redirected to the sensei-pro page

## Screenshots

![Capture d’écran 2023-09-26 à 16 27 52](https://github.com/Automattic/wp-calypso/assets/1420594/86ba4f24-bb28-4578-8db3-4a73cd591175)
![Capture d’écran 2023-09-26 à 16 28 02](https://github.com/Automattic/wp-calypso/assets/1420594/317a2c15-a366-46fb-9ebd-438bc7ffa83b)
